src/util/ScreenSaver/ScreenSaver.h

```diff
@@ -0,0 +1,126 @@
+#include "Cemu/Logging/CemuLogging.h"
+
+#if BOOST_OS_MACOS
+#include <IOKit/pwr_mgt/IOPMLib.h>
+#endif
+
+#if BOOST_OS_LINUX
+#include <spawn.h>
+#ifdef USE_ELOGIND
+#include <elogind/sd-bus.h>
+#include <elogind/sd-daemon.h>
+#else
+#include <systemd/sd-bus.h>
+#include <systemd/sd-daemon.h>
+#endif
```

Comment:
Instead of using sd-bus, I think it'd be better to use plain [libdbus](https://dbus.freedesktop.org/doc/api/html/). It is a bit less developer-friendly, but it is more widely available (not Linux-only) and not part of the systemd project, which some users may want to avoid for whatever reason; here we're also only making a couple of dbus calls, so it shouldn't be that big of a pain.

Reply:
I will try my best, but DBus is outside of my area of expertise, which is why I adapted implementations from other software.

Reply:
The one thing I'm confused about is how to connect to the user bus using libdbus like sd-bus does. I haven't been able to find documentation about this.

Reply:
I think libdbus calls it DBUS_BUS_SESSION. Found an example program that uses libdbus:
https://github.com/wware/stuff/blob/master/dbus-example/dbus-example.c
You'll problably need to do something similar to the query function in this example.